Library/ViewModels/PledgeViewModel.swift

```diff
@@ -219,8 +223,37 @@ public class PledgeViewModel: PledgeViewModelType, PledgeViewModelInputs, Pledge
         PKPaymentAuthorizationViewController.merchantIdentifier
       ) as PaymentAuthorizationData }
 
-    self.goToApplePayPaymentAuthorization = paymentAuthorizationData
+    let goToApplePayPaymentAuthorization = pledgeAmountIsValid
+      .takeWhen(self.applePayButtonTappedSignal)
+      .filter { isTrue($0) }
+
+    let showApplePayAlert = pledgeAmountIsValid
       .takeWhen(self.applePayButtonTappedSignal)
+      .filter { isFalse($0) }
```

Comment:
Can be `filter(isFalse)`.

Reply:
😄 and dis?

Reply:
![gif](https://media.giphy.com/media/UY5NnctabOECc/giphy.gif)